src/system.js

```diff
@@ -14,16 +14,23 @@ contexts.forEach(context => {
   context.keys().forEach(key => components.push(context(key).default))
 })
 
+function initializeCustomProps(tokens, prefix) {
+  for (const param in tokens) {
```

Comment:
Could you add an early return above the for loop, checking if tokens are provided?